py/src/braintrust/framework.py

```diff
@@ -158,6 +158,13 @@ class Evaluator:
     """
     metadata: Optional[EvalMetadata]
 
+    """
+    The number of times to run the evaluator per input. This is useful for evaluating applications that
+    have non-deterministic behavior and gives you both a stronger aggregate measure and a sense of the
+    variance in the results.
+    """
+    trial_count: int = 1
```

Comment:
nit: for uniformity with TS maybe we could keep this None and default it to 1 in the implementation? It's sad that python doesn't provide a `coalesce` function, but I have defined one here https://github.com/braintrustdata/braintrust/blob/e7f1ea81c43443c2c63e364d6dd8542e3de57fac/api/chalicelib/util.py#L38.

Reply:
I think I prefer this because it'll be nicer if someone jumps into the definition and is poking around